ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -18,12 +18,53 @@ periodics:
     labels:
       master: ci.openshift.redhat.com
     name: azure_build_node_image_centos
-- agent: jenkins
+- agent: kubernetes
   cron: ""
+  decorate: true
   interval: 24h
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-azure-secret
+      - --template=/usr/local/build-base-image
+      - --target=build-base-image
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: release-3.10.yaml
+            name: ci-operator-openshift-openshift-ansible
+      - name: JOB_NAME_SAFE
+        value: build-base-image
+      - name: CLUSTER_TYPE
+        value: azure
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources: {}
+      volumeMounts:
+      - mountPath: /usr/local/build-base-image
+        name: job-definition
+        subPath: build-base-image.yaml
```

Comment:
Is this file name correct?

https://github.com/openshift/openshift-ansible/blob/master/playbooks/azure/openshift-cluster/build_base_image.yml

Reply:
I think this refers to the location of the template to use for ci-operator. The node build image section specifies `build-node-image`. @kargakis can you confirm?

Reply:
Ok, makes sense.

Reply:
Yes, this is correct.